R/dash.R

```diff
@@ -255,6 +254,9 @@ Dash <- R6::R6Class(
           }
         }
 
+        # set the callback context associated with this invocation of the callback
+        private$callback_context_ <- setCallbackContext(request$body)
```

Comment:
This looks great. I think all you need to do to finish off the callback context part of this PR is set it back to `NULL` after the `do.call` below, and have `callback_context` throw an error if it finds `NULL`.

Reply:
fixed in 6fc1cac2e3506ba67ead79b496b51127ba7314e0

Reply:
Oh sorry - what I meant was, down here when you access `callback_context_`

https://github.com/plotly/dashR/pull/87/files#diff-1881af3720f8f49fd3a44e51c9fb5a0dR489

At that point, if it's `NULL`, that means you're not in a callback so you shouldn't be able to access it, so throw an error.

Reply:
fixed in 96e4672